modules/Search/components/Hit.tsx

```diff
@@ -40,35 +60,32 @@ function HitComponent({ hit }: Props) {
             <div className={cardStyles.content}>
                 {categories.length > 0 && (
```

Comment:
Shouldn't this check for `displayedCategories.length` instead? As we're passing `displayedCategories` below.

```suggestion
                {displayedCategories.length > 0 && (
```

Reply:
Good catch! Thanks! :+1:

Reply:
e06f8a96490901da6ae3cca3220cc0cbad04c5d2